### PR TITLE
Ports previous changes of breasts to the penis

### PR DESCRIPTION
**Pull request by jbox144**
## About The Pull Request

Adds all my changes to the breasts to the penis, meaning penis production now scales with organ size, penises can have their fluid type changed, e.t.c.

## Why It's Good For The Game

🅱️enis

## Changelog
:cl:
add: Added the ability to customise the fluid your penis produces
add: Added fluid production scaling to penises
/:cl:

### CODE REVIEW

File: hyperstation/code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/datum/crafting_recipe/milker
```
Comment (improvedname) [inline hyperstation/code/modules/crafting/recipes.dm]: It'd probaly be better if you call it /datum/crafting_recipe/milking_machine

File: hyperstation/code/modules/crafting/recipes.dm [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	name = "Milking Machine"
+	reqs = list(/obj/item/stack/cable_coil = 5, /obj/item/stack/rods = 2, /obj/item/stack/sheet/cardboard = 1, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/stock_parts/manipulator = 1)
+	result = /obj/item/milker
```
Comment (improvedname) [inline hyperstation/code/modules/crafting/recipes.dm]: Same for the path /obj/item/milking_machine

File: hyperstation/code/modules/crafting/recipes.dm [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	tools = list(TOOL_WELDER, TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	category = CAT_MISC
+
+/datum/crafting_recipe/milker/penis
+	name = "Penis Milking Machine"
+	reqs = list(/obj/item/stack/cable_coil = 5, /obj/item/stack/rods = 1, /obj/item/stack/sheet/cardboard = 1, /obj/item/reagent_containers/glass/beaker/large = 1, /obj/item/stock_parts/manipulator = 1)
+	result = /obj/item/milker/penis
```
Comment (improvedname) [inline hyperstation/code/modules/crafting/recipes.dm]: /obj/item/milking_machine/penis

File: hyperstation/code/modules/crafting/recipes.dm [hunk 4]
```diff
@@ -0,0 +1,11 @@
+/datum/crafting_recipe/milker
+	name = "Milking Machine"
+	reqs = list(/obj/item/stack/cable_coil = 5, /obj/item/stack/rods = 2, /obj/item/stack/sheet/cardboard = 1, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/stock_parts/manipulator = 1)
+	result = /obj/item/milker
+	tools = list(TOOL_WELDER, TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	category = CAT_MISC
+
+/datum/crafting_recipe/milker/penis
```
Comment (improvedname) [inline hyperstation/code/modules/crafting/recipes.dm]: /datum/crafting_recipe/milking_machine/penis

File: hyperstation/code/obj/milking machine.dm [hunk 5]
```diff
@@ -0,0 +1,83 @@
+/obj/item/milker
```
Comment (improvedname) [inline hyperstation/code/obj/milking machine.dm]: /obj/item/milking_machine

File: hyperstation/code/obj/milking machine.dm [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+			if (O.reagents.total_volume >= transfer_rate * 2)
+				if (inserted_item.reagents.total_volume < inserted_item.reagents.maximum_volume)
+					O.reagents.trans_to(inserted_item.reagents, amount = transfer_rate)
+				else
+					to_chat(W, "<span class='notice'>[src] stops pumping. [inserted_item] is full.</span>")
+					on = FALSE
+					STOP_PROCESSING(SSobj, src)
+					UpdateIcon()
+
+/obj/item/milker/penis
```
Comment (improvedname) [inline hyperstation/code/obj/milking machine.dm]: /obj/item/milking_machine/penis

**Comment by jbox144** [comment]
> Okay I'm glad you fixed your stuff as well (in a different PR from #237?) but you also have Tania station stuff in here.
> Self insert stuff is really a no-go, and you didn't even list it in the changelog here either.

*innocent whistling*

Alright, I'll remove it

**Pull request merged**